modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -994,9 +997,13 @@ def obj.class
             VCR.use_cassette('evss/disability_compensation_form/form_526_invalid_validation') do
               VCR.use_cassette('brd/countries') do
                 VCR.use_cassette('bgs/claims/claims') do
-                  post path, params: data, headers: headers.merge(auth_header)
-                  parsed = JSON.parse(response.body)
-                  expect(parsed['errors'].size).to eq(2)
+                  VCR.use_cassette('claims_api/v1/disability_comp/bd_token') do
+                    VCR.use_cassette('claims_api/v1/disability_comp/invalid') do
```

Comment:
if we're using the new cassette here, we could remove the old one so we don't have that additional `.use_cassette`